st2actions/st2actions/container/base.py

```diff
@@ -327,28 +321,26 @@ def _get_rerun_reference(self, context):
         execution_id = context.get('re-run', {}).get('ref')
         return ActionExecution.get_by_id(execution_id) if execution_id else None
 
-    def _get_runner(self, runnertype_db, action_db, liveaction_db):
-        resolved_entry_point = self._get_entry_point_abs_path(action_db.pack,
-                                                              action_db.entry_point)
+    def _get_runner(self, runner_type_db, action_db, liveaction_db):
+        resolved_entry_point = self._get_entry_point_abs_path(action_db.pack, action_db.entry_point)
         context = getattr(liveaction_db, 'context', dict())
         user = context.get('user', cfg.CONF.system_user.user)
+        config = None
 
         # Note: Right now configs are only supported by the Python runner actions
-        if runnertype_db.runner_module == 'python_runner':
-            LOG.debug('Loading config for pack')
-
+        if runner_type_db.runner_module == 'python_runner':
+            LOG.debug('Loading config from pack for python runner.')
             config_loader = ContentPackConfigLoader(pack_name=action_db.pack, user=user)
             config = config_loader.get_config()
-        else:
-            config = None
 
-        runner = get_runner(package_name=runnertype_db.runner_package,
-                            module_name=runnertype_db.runner_module,
-                            config=config)
+        runner = get_runner(
+            package_name=runner_type_db.runner_package,
+            module_name=runner_type_db.runner_module,
+            config=config)
 
         # TODO: Pass those arguments to the constructor instead of late
         # assignment, late assignment is awful
-        runner.runner_type_db = runnertype_db
+        runner.runner_type = runner_type_db
```

Comment:
I really dislike not having a `_db` suffix since it's confusing.

If anything, I would to the reverse - add `_db` suffix to all the objects which represent DB models so it's intermediately clear what you are working with.

Reply:
I agree if we go back and make this consistent across the code base.  Going to take me some time before I get back to this.  Maybe we can give this task to the new guy?